lib/fastly/lib/index.js

```diff
@@ -9,16 +9,17 @@
  * Dependencies
  */
 const request = require('request');
-const Q = require('q');
 const debug = require('debug')('fastly');
 
 /**
  * Constructor
  */
-function Fastly (apikey, service, options) {
+function Fastly (apikeys, service, options) {
 
 	var opts = options || {};
-	this.apikey = apikey || '';
+	this.apikey = apikeys.shift();
+	this.apikeys = apikeys || [];
```

Comment:
again, it's an internal class, and the task that calls it takes care of ensuring it's an array... but I supose there's no harm in adding this